### PR TITLE
:tada: Add openvasxml parser #8761

### PR DESCRIPTION
**Pull request by manuel-sommer**
See linked Issue.

### CODE REVIEW

**Comment by manuel-sommer** [comment]
Hi @Maffooch, 
why is this PR failing?

**Comment by mtesauro** [comment]
@manuel-sommer I thought it might the the chrome / chrome driver issue that cropped up earlier today but it looks like it's something else.

I went ahead and re-kicked the tests to see if it was a transitory issue or something else.

**Comment by mtesauro** [comment]
@manuel-sommer Yup, it's the chrome/chrome-driver issue. There's a PR to fix this happening now.

**Comment by manuel-sommer** [comment]
Thank you for investigating

**Comment by mtesauro** [comment]
@manuel-sommer OK, we got the Github actions happy again. Mind rebasing so those tests will run correctly (and pass)

**Comment by manuel-sommer** [comment]
@mtesauro, @Maffooch, @devGregA ready to review.

**Comment by Maffooch** [comment]
Something is up with the commit history here. Please take a look

**Comment by manuel-sommer** [comment]
Now, everything shoule be fine @Maffooch. Let's squash the commits when we merge this MR

File: dojo/tools/openvas_xml/parser.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        root = tree.getroot()
+        if "report" not in root.tag:
+            raise NamespaceErr(
+                "This doesn't seem to be a valid Greenbone OpenVAS xml file."
+            )
+        report = root.find("report")
+        results = report.find("results")
+        for result in results:
+            for finding in result:
+                print(finding)
```
Comment (dwalleck) [inline dojo/tools/openvas_xml/parser.py:41]: Is this line for debugging purposes or is it necessary for the parser?

File: dojo/tools/openvas_xml/parser.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                if finding.tag == "name":
```
Comment (dwalleck) [inline dojo/tools/openvas_xml/parser.py:42]: This is my personal preference, but when I have more than three cases to match on, I find using `match` to be easier to understand

Comment (manuel-sommer) [reply dojo/tools/openvas_xml/parser.py:41]: For me both would be fine. If the maintainers do not like it my way and therefore will not accept the PR, I can do it.

**Review by Maffooch — APPROVED** [review]

**Comment by manuel-sommer** [comment]
@mtesauro please review

File: dojo/tools/openvas_xml/parser.py (hunk 2 above)
Comment (manuel-sommer) [reply dojo/tools/openvas_xml/parser.py:41]: I have the first approval, so I am closing this.

**Review by mtesauro — APPROVED** [review]
Approval

**Comment by manuel-sommer** [comment]
@blakeaowens and @devGregA please approve

**Review by blakeaowens — APPROVED** [review]

**Comment by manuel-sommer** [comment]
@cneill could you please approve this PR?

**Comment by manuel-sommer** [comment]
@damiencarol could you approve this PR?

**Review by grendel513 — APPROVED** [review]

**Comment by manuel-sommer** [comment]
Can you merge it @grendel513 ?

**Pull request merged**